sandboxed_containers/upgrade-sandboxed-containers.adoc

```diff
@@ -8,6 +8,23 @@ toc::[]
 
 You can upgrade the components of {sandboxed-containers-first} by upgrading {sandboxed-containers-operator} and {sandboxed-containers-first} artifacts.
 
+[IMPORTANT]
+====
+When upgrading OpenShift Container Platform 4.10 with OpenShift sandboxed containers 1.2 to OpenShift Container Platform 4.11 with OpenShift sandboxed containers 1.3, the recommended path is first to upgrade OpenShift sandboxed containers to 1.3, and then to upgrade OpenShift Container Platform to to 4.11.
+
+After the upgrade of OpenShift sandboxed containers, new features and bug fixes in the Operator code are available immediately. However, additional steps are required to upgrade certain resources.
+
+* To get the new features and bug fixes for the Kata runtime, delete and re-create the `KataConfig` resource. This must be performed after the cluster upgrade, to deploy the new RHCOS extension (otherwise the old Kata RPMs will be used), and prior to the OpenShift Container Platform upgrade, because the Kata runtime is shipped as an RHCOS extension.
+
+* To get the new features and bug fixes for the monitor pod, manually patch the `KataConfig` resource. To do so, either edit the `KataConfig` YAML in the Web console, or run the following CLI command:
++
+[source,terminal]
+----
+$ oc patch kataconfig example-kataconfig --type merge --patch '{"spec":{"kataMonitorImage":"registry.redhat.io/openshift-sandboxed-containers/osc-monitor-rhel8:1.3.0"}}'
+----
+This step is unnecessary if you recreated `KataConfig`.
```

Comment:
Since we aren't asking the user to delete/re-create the `KataConfig` anymore, we could maybe just drop this last sentence ?

Reply:
This still needs to be done after the OCP upgrade.